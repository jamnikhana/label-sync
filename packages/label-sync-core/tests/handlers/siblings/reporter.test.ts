describe('sibling sync reporter', () => {
  test.skip('correctly reports', async () => {
    return
  })
})
