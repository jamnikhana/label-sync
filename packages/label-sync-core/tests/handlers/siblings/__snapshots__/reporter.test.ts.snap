// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sibling sync reporter correctly generates report 1`] = `
"maticzav/label-sync
This is an autogenerated report for your project.
(dry run: "false")


Hey ✌️ we synced issues in your repository. Check the changes we made:

Another Issue Title
Issue number: 5

Added basic, kind/bug."
`;

exports[`sibling sync reporter generates correct empty report 1`] = `
"maticzav/label-sync
This is an autogenerated report for your project.
(dry run: "true")


Hey ✌️ we synced issues in your repository. Check the changes we made:

"
`;
